app/assets/javascripts/lajoya.js

```diff
@@ -72,30 +60,39 @@
 
   function createDownloadLink() {
     recorder && recorder.exportWAV(function(blob) {
+      if(blob.size > 70000 && blob.size < 2500000) {
       var url = URL.createObjectURL(blob);
       var li = document.createElement('li');
       var au = document.createElement('audio');
-      var clipID = new Date().toISOString() + '.wav';
-      au.preload='auto'
-      au.className = "clip"
+      au.preload='auto';
+      au.className = "clip";
       au.src = url;
       li.appendChild(au);
       recordingslist.appendChild(li);
+    } else { alert("Clip tiene que ser entre 1 y 30 segundos")};
```

Comment:
Mixed double and single quotes.<br>Missing semicolon.<br>Unnecessary semicolon.<br>'alert' is not defined.